README.md

```diff
@@ -10,19 +9,14 @@ Network Service Mesh Site
 
 
 This is the initial Network Service Mesh website. It was created using
-[Hugo](https://gohugo.io/) with the [Airspace Hugo theme](https://themes.gohugo.io/airspace-hugo/).
-
-NOTE: The theme has been added as a git submodule, so to ensure this is checked out
-when you clone this repository, run your clone as follows:
-
-```
-git clone --recurse-submodules https://github.com/networkservicemesh/site.git
-```
+[Hugo](https://gohugo.io/).
 
 To quickly install Hugo, follow [this link](https://gohugo.io/getting-started/quick-start/).
 
+You will also need to install [Yarn](https://yarnpkg.com/en/).  Easy install instructions are [here](https://yarnpkg.com/en/docs/install#mac-stable)
```

Comment:
Can I suggest: https://yarnpkg.com/en/docs/install#debian-stable as a neutral default.

Reply:
Or better yet:  https://yarnpkg.com/en/docs/install :)

Reply:
Fixed in update :)